test/detailed-results-4-tabs-MD010.results.json

```diff
@@ -0,0 +1,29 @@
+[
+  {
+    "errorContext": null,
+    "errorDetail": "Column: 1",
+    "errorRange": [1, 13],
+    "lineNumber": 1,
+    "ruleDescription": "Hard tabs",
+    "ruleInformation": "https://github.com/DavidAnson/markdownlint/blob/v0.23.1/doc/Rules.md#md010",
```

Comment:
Please update the version to "v0.0.0" in these places so they aren't outdated by version upgrades.